CHANGES.md

```diff
@@ -4,13 +4,23 @@ This file is only intended to serve the developers of this repository and of the
 
 This provides an overview of the releases that have been made using this release pipeline.
 
-## Work In Progress - March: 2023: Security release 1.39
+## Work In Progress - April: 2023: 1.39.3 Security release
 
-For a MediaWiki update to 1.39.2
+- [MediaWiki security release announcement](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/message/6UQBHI5FWLATD7QO7DI4YS54U7XSSLAN/)
+- [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.39)
 
 | Suite Version | MediaWiki release | Date available | Run number | Release task  |
 |---------------|-------------------|----------------|------------| --------------|
-| wmde.12 ([spec](),[metadata]()) | 1.39.2    | March 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
+| wmde.13 ([spec](),[metadata]()) | MediaWiki (1.39.3, PHP 8.0, PHP 8.1, PHP 8.2) | April 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
+
+## Work In Progress - April: 2023: 1.38.6 Security release
```

Comment:
In the past we used a single heading for a single set of security releases
So the heading for wmde.13 should be joined with wmde.12
Together these make up the April suite security release.

Reply:
will join them then! :)